test/box-luatest/gh-5602-environment-vars-cfg_test.lua

```diff
@@ -0,0 +1,198 @@
+local t = require('luatest')
+local fio = require('fio')
+
+local justrun = require('luatest.justrun')
+local treegen = require('luatest.treegen')
+
+local g = t.group()
+
+local tmpdir = fio.tempdir()
+local listen_sock1 = fio.pathjoin(tmpdir, 'listen1.sock')
+local listen_sock2 = fio.pathjoin(tmpdir, 'listen2.sock')
+
+-- Check that environment cfg values are set correctly.
+g.test_gh_5602_tarantool_env_variables_1 = function()
+    local dir = treegen.prepare_directory({}, {})
+    local script_name = 'test_gh_5602_tarantool_env_variables_1.lua'
+    local script_body = ([[
+local test = require('tap').test('gh-5602')
```

Comment:
BTW, if you're interested, I have a helper that hide some technics to run luatest-alike test cases as separate scripts.

https://github.com/tarantool/tarantool/blob/65827458edb1eccd92ffeeee1a57a55e05236ac0/test/config-luatest/box_cfg_env_test.lua#L102-L110